content/docs/guides/koyeb.md

```diff
@@ -0,0 +1,132 @@
+
+```bash
+postgres://<user>:<password>@<host>:<port>/neondb
+```
+
+Store this value in a safe place. It is required later. The connection string specifies `neondb` as the database. This is the default database created with each Neon project. You will use this database with the example application.
+
+## Deploy the application on Koyeb
+
+You can deploy on Koyeb using the control panel or the Koyeb CLI.
+
+### From the Koyeb control panel
+
+To deploy the application from the Koyeb [control panel](https://app.koyeb.com/), follow these steps:
+
+1. Select **Create App**.
+1. Select GitHub as the deployment method.
+1. Enter `https://github.com/koyeb/example-express-prisma` in the **Public GitHub repository** field.
+1. Keep `example-express-prisma` as the name and `main` as the branch.
+1. In **Build and deployment settings**, enable the **Override** setting and add the following **Build command**: `npm run postgres:init`
+1. Select the region closest to your Neon database.
+1. Under **Advanced** > **Environment variables**, add a `DATABASE_URL` environment variable to enable the application to connect to your Neon PostgreSQL database. Set the value to the Neon connection string provided to you when you created the Neon project.
+1. Enter a name for your app. For example, `express-neon`
+1. Click **Deploy**.
+
+Koyeb builds the application. Once the build has finished, you can access your application running on Koyeb by clicking the URL ending with `.koyeb.app`.
+
+The example application exposes a `/planets` endpoint that you can use to list planets from the database. Once your deployment is live, you should see the following results when navigating to `https://<YOUR_APP_URL>.koyeb.app/planets`:
+
+```json
+[
+{
+    "id": 1,
+    "name": "Mercury"
+},
+{
+    "id": 2,
+    "name": "Venus"
+},
+{
+    "id": 3,
+    "name": "Mars"
+}
+]
+```
+
+### From the Koyeb CLI
+
+The [Koyeb CLI](https://www.koyeb.com/docs/quickstart/koyeb-cli) requires an API access token, which you can generate in the Koyeb [control panel](https://app.koyeb.com/), under **Account** > **API**.
+
+To deploy the example application using the Koyeb CLI, run the following command in your terminal.
+
+```bash
+koyeb app init express-neon \
+--git github.com/koyeb/example-express-prisma \
+--git-branch main \
+--git-build-command "npm run postgres:init" \
+--ports 8080:http \
+--routes /:8080 \
+--env PORT=8080 \
+--env DATABASE_URL="postgres://<user>:<password>@<host>:5432/neondb"
+```
+
+Make sure to replace `<user>`, `<password>`, and `<host>` with your Neon connection string values.
+
+### Access Koyeb deployment logs
+
+To track the app deployment and visualize build logs, execute the following command:
+
+```bash
+koyeb service logs express-neon/express-neon -t build
+```
+
+### Access your app
+
+Once the deployment of your application has finished, you can retrieve the public domain to access your application by running the following command:
```

Comment:
Once the build and deployment have finished, you can retrieve the public domain to access your application by running the following command:

Reply:
Thank you. I will address this in another PR.